### DISCUSSION TITLE
esp32/Arduino/02.esp32-arduino-install

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#10] [General]
# esp32/Arduino/02.esp32-arduino-install

https://docs.geeksman.com/esp32/Arduino/02.esp32-arduino-install.html

**Comment by CherishNA** [comment]
来了老哥，

**Comment by szthor** [comment]
非常喜欢这位作者的做事细致风格

**Comment by wwwqdykkjcom-atm** [comment]
为什么没有我的Arduino IDE开发板中 没有ESP32 的 2.0.4版本  ？   安装个ESP32 开发板太难了

**Comment by luqqqqqqq** [comment]
太棒了老哥😀，小的时候玩arduino的好多坑都看到了😥

**Reply by 248523352** [reply]
请问资料包在哪儿下载

**Comment by luqqqqqqq** [comment]
抱歉，我不确定你的意思。文章中的arduino可以在官网下载，esp开发包是通过添加json链接，在arduino内下载的，对于下载慢的情况可以开VPN解决。如果没有VPN，文章作者提供了解决方法二。此处我给出一个镜像链接espressif_arduino-esp32






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "bigrich-luo/DocsGiscus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月23日(星期三) 上午10:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [bigrich-luo/DocsGiscus] esp32/Arduino/02.esp32-arduino-install (Discussion #10)





 
请问资料包在哪儿下载

**Comment by codeskyblue** [comment]
我用的镜像
https://arduino.me/packages/esp32.json
然后version这里选择 3.2.1-cn 就行了。这个-cn后面才知道原来是国内镜像版本的意思。
